### ISSUE TITLE
Cannot read property `toJSDate` of undefined

### ISSUE THREAD

**Issue by CityVibes** [Purchasely/Purchasely-Firebase-Extension#51]
#### Issue:
Subscriptions are not being updated in Firestore or Claims. Same sandbox subscriptions are properly reflected in the Purchasely dashboards.

#### Extension version:
`purchasely/purchasely-in-app-purchases@1.0.4`

#### Notes
I inspected extension function logging and noticed several error logs.

##### Stacktraces:
```
TypeError: Cannot read property 'toJSDate' of undefined\n    at Object.create (/workspace/lib/v3/purchasely-events/repository.js:12:91)\n    at Object.create (/workspace/lib/v3/purchasely-events/service.js:7:42)
```
also
```
Error: Process exited with code 16
    at process.<anonymous> (/layers/google.nodejs.functions-framework/functions-framework/node_modules/@google-cloud/functions-framework/build/src/invoker.js:92:22)
    at process.emit (events.js:400:28)
    at process.emit (domain.js:475:12)
    at process.exit (internal/process/per_thread.js:179:15)
    at sendCrashResponse (/layers/google.nodejs.functions-framework/functions-framework/node_modules/@google-cloud/functions-framework/build/src/logger.js:44:9)
    at process.<anonymous> (/layers/google.nodejs.functions-framework/functions-framework/node_modules/@google-cloud/functions-framework/build/src/invoker.js:88:44)
    at process.emit (events.js:400:28)
    at process.emit (domain.js:475:12)
    at processPromiseRejections (internal/process/promises.js:245:33)
    at processTicksAndRejections (internal/process/task_queues.js:96:32)
```

**Comment by ste26054** [comment]
Thanks for reporting this issue, we are looking into it!

**Comment by ste26054** [comment]
Hi @CityVibes, we released version 1.0.5 of the extension fixing the bug above, could you please update and tell us if this update fixed the problem? Thanks!

**Comment by CityVibes** [comment]
Hey, yeah this seems to fix the issue. Thank you folks.
